doc/api/modules.md

```diff
@@ -598,11 +598,26 @@ filename scales linearly with the number of registered extensions.
 In other words, adding extensions slows down the module loader and
 should be discouraged.
 
-#### require.resolve()
+#### require.resolve([options])
```

Comment:
Should this be something like `#### require.resolve(name[, options])`?

Reply:
+ It seems till we automate this, we should check possible links in all docs if wi change headers. Currently, we have [one for this header](https://github.com/nodejs/node/blame/35deef06765cdf5638a7d031ab7e832af3cf4c82/doc/api/process.md#L1903), so it also needs updating.

Reply:
Thanks for pointing this out. I've updated the documentation and searched for relevant links.